spec/graphql/mutations/create_organization_mutation_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'createOrganization mutation' do
+  let!(:current_user) { create :user }
+  let(:organization_data) do
+    {
+      'name' => 'foobar',
+      'displayName' => 'Foobar',
+      'description' => 'This is the foobar',
+    }
+  end
+
+  let(:context) { {current_user: current_user} }
+  let(:variables) { {'data' => organization_data} }
+
+  let(:result) do
+    OntohubBackendSchema.execute(
+      query_string,
+      context: context,
+      variables: variables
+    )
+  end
+
+  let(:query_string) do
+    <<-QUERY
+    mutation CreateOrganization($data: NewOrganization!) {
+      createOrganization(data: $data) {
+        id
+        displayName
+        description
+        members {
+          id
+        }
+      }
+    }
+    QUERY
+  end
+
+  context 'Successful creation' do
+    subject { result }
```

Comment:
Why don't you put the `subject` outside the `context`?

The same question applies to the other specs.